wyvern/components/features/feature_logger.py

```diff
@@ -75,21 +78,26 @@ def feature_event_generator():
                 A list of feature events.
             """
             timestamp = datetime.utcnow()
+
+            # Extract column names excluding "IDENTIFIER"
+            feature_columns = input.feature_df.df.columns[1:]
+
             return [
                 FeatureEvent(
                     request_id=input.request.request_id,
                     run_id=run_id,
                     api_source=url_path,
                     event_timestamp=timestamp,
                     event_data=FeatureLogEventData(
-                        feature_identifier=feature_data.identifier.identifier,
-                        feature_identifier_type=feature_data.identifier.identifier_type,
-                        feature_name=feature_name,
-                        feature_value=feature_value,
+                        # todo (kerem): What about composite identifiers? We only get the primary identifier here.
+                        feature_identifier_type=row[IDENTIFIER].split(":")[0],
+                        feature_identifier=row[IDENTIFIER].split(":")[1],
```

Comment:
if the identifier is a composite identifier, is the primary identifier actually being logged?